app/file.go

```diff
@@ -439,6 +439,11 @@ func (a *App) UploadFile(data []byte, channelId string, filename string) (*model
 	if appError != nil {
 		return nil, appError
 	}
+	_, err := a.GetChannel(channelId)
```

Comment:
Should we run this check at the top of the method? It seems like the file has already been uploaded at this point.

Reply:
You're right, I completely missed this in my review! @waqasraz would you be open to move this at the top of the method?